test/bijectors/leaky_relu.jl

```diff
@@ -0,0 +1,86 @@
+using Test
+
+using Bijectors
+using Bijectors: LeakyReLU
+
+using LinearAlgebra
+using ForwardDiff
```

Comment:
I think my only real comment here is on testing other AD backends like ReverseDiff, but I'm not sure how important that is here.

Reply:
There are some tests for it in `test/bijectors/interface.jl`. But yeah, testing in Bijectors is honestly a bit of mess atm. In a couple of the other PRs I've added some functionality which makes it easier to use a "standardized" testing suite for a new `Bijector`, so the plan is to use that in the future :+1: